product/src/main/java/com/orderService/product/controller/ProductController.java

```diff
@@ -0,0 +1,20 @@
+package com.orderService.product.controller;
+
+import com.zorder.orderservice.entity.Product;
+import com.zorder.orderservice.services.Impl.ProductServiceImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@CrossOrigin
+@RequestMapping("/api")
+public class ProductController {
+
+    @Autowired
+    ProductServiceImpl productServiceImpl;
+
+    @PostMapping("/v1/getOrder")
```

Comment:
PostMapping ? for fetching?, use @GetMapping with name as query param

Reply:
use noun "/api/v1/order" rather than "/api/v1/getOrder"  "get order" action should be reflected by GET (HTTP verb)